tox.ini

```diff
@@ -21,3 +21,9 @@ commands =
 
 [testenv:cov-report]
 commands = py.test --cov=cookiecutter --cov-report=term --cov-report=html
+
+[testenv:mypy]
+deps =
+    mypy-lang
+# Not all imports have stubs yet, so let's ignore those for now.
+commands = mypy cookiecutter --silent-imports
```

Comment:
The `--silent-imports` argument is to silence type errors that come form importing non-annotated code. It's also possible to put a `# type: ignore` on each of the offending import lines, but doing it this way makes it easier to throw a switch at runtime and see if anything has changed.

Reply:
Closing as this is far too old and also has a weird implementation. Enabling mypy can be done as part of the flake8, and not with extra tox environments. 

If anyone wants to restart the challenge, I will be happy to help.